src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/CreateTableOperation.java

```diff
@@ -62,6 +66,10 @@ public Uni<Supplier<CommandResult>> execute(
     CqlIdentifier tableIdentifier = CqlIdentifier.fromInternal(tableName);
     CreateTableStart create = createTable(keyspaceIdentifier, tableIdentifier).ifNotExists();
 
+    // Add if not exists flag based on request
+    if (ifNotExists) {
+      create = create.ifNotExists();
```

Comment:
Isn't this already being set above (line 63/67)?

Reply:
Fixed it.